File: app/src/main/java/com/jonguk/androidreviews/java/recycler/RecyclerViewHolder.java
package com.jonguk.androidreviews.java.recycler;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.jonguk.androidreviews.R;

/**
 * Created by Jonguk on 2017. 10. 2..
 */

class RecyclerViewHolder extends RecyclerView.ViewHolder {

    private TextView mTextView;

    RecyclerViewHolder(View itemView) {
        super(itemView);
        mTextView = (TextView) itemView.findViewById(R.id.recycler_text);
    }

    void bind(String text) {
        mTextView.setText(text);
    }

}

File: app/src/main/java/com/jonguk/androidreviews/kotlin/recycler/RecyclerViewHolder.kt
package com.jonguk.androidreviews.kotlin.recycler

import android.support.v7.widget.RecyclerView
import android.view.View
import kotlinx.android.synthetic.main.item_recycler.view.*

/**
 * Created by Jonguk on 2017. 9. 24..
 */
class RecyclerViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    fun bind(text: String) {
        itemView.recycler_text.text = text
    }
}